src/_codux/boards/dropdown-menu/dropdown-menu.board.tsx

```diff
@@ -0,0 +1,19 @@
+import { createBoard } from '@wixc3/react-board';
+import { HomeDropdownMenu } from '../../../components/dropdown-menu/home-dropdown-menu';
+import { StoreContextProvider } from '../../../data-management/store-context-provider';
+import { FakeDataService } from '../../../data-management/fake-data/fake-data-service';
+
+export default createBoard({
+    name: 'ContextMenu - Home screen',
+    Board: () => {
+        const storageService = new FakeDataService([]);
```

Comment:
Add a note with id `1` to your fake storage, otherwise menu actions break the board.